tests/test_sandbox_scripts.py

```diff
@@ -76,8 +76,10 @@ def __call__(self):
                     compile(open(self.filename).read(), self.filename, 'exec'),
                     global_dict)
             except (ImportError, SyntaxError):
-                print(traceback.format_exc())
-                raise AssertionError("%s cannot be exec'd" % (self.filename,))
+                print(unicode(traceback))
```

Comment:
Can we use `print("{0}".format(err))` here? ([source][1])

[1]: https://docs.python.org/3/tutorial/errors.html#handling-exceptions